### PR TITLE
Change the output

### PR DESCRIPTION
**Pull request by Nosferatul**

### CODE REVIEW

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -63,7 +63,8 @@ def highlight_text(no_color, text, lexer=TextLexer(), style=None):
         formatter = TerminalTrueColorFormatter(style=NoColorStyle)
     return highlight(text, lexer, formatter)[:-1]
 
-
+STR_VAL_SPACES = ('Spaces are to be used instead of tabs? (True/False)')
+STR_VAL_LANG = ('Enter the programming language of the source code:')
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: think should be a space after ":"

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -571,6 +575,7 @@ def choose_action(console_printer, actions):
             line = format_lines(STR_ENTER_NUMBER)
 
             choice = input(line)
+            print("ASDASD" + choice)
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: thats beautiful, so poetic.

File: coalib/output/ConsoleInteraction.py [hunk 3]
```diff
@@ -565,22 +565,31 @@ def choose_action(console_printer, actions):
...
-
+            if action.desc == '(O)pen file':
+                possible_actions.insert(i, 'O')
+            elif action.desc == '(A)pply patch':
+                possible_actions.insert(i, 'A')
+            elif action.desc == 'Print (M)ore info':
+                possible_actions.insert(i, 'M')
+            elif action.desc == 'Add (I)gnore comment':
+                possible_actions.insert(i, 'I')
+            elif action.desc == 'Print d(E)bug message':
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: shouldnt debug be a D?

File: coalib/results/result_actions/PrintDebugMessageAction.py [hunk 4]
```diff
@@ -15,7 +15,7 @@ def is_applicable(result: Result, original_file_dict, file_diff_dict):
 
     def apply(self, result, original_file_dict, file_diff_dict):
         """
-        Print debug message
+        Print (D)ebug message
```
Comment (Adrianzatreanu) [inline coalib/results/result_actions/PrintDebugMessageAction.py]: heres a D, upper you said E here i think

File: coalib/output/ConsoleInteraction.py (hunk 1 above)
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: bump

File: coalib/output/ConsoleInteraction.py [hunk 5]
```diff
@@ -566,21 +566,31 @@ def choose_action(console_printer, actions):
...
+            if action.desc == '(O)pen file':
+                possible_actions.insert(i, 'O')
+            elif action.desc == '(A)pply patch':
+                possible_actions.insert(i, 'A')
+            elif action.desc == 'Print (M)ore info':
+                possible_actions.insert(i, 'M')
+            elif action.desc == 'Add (I)gnore comment':
+                possible_actions.insert(i, 'I')
+            elif action.desc == 'Print D(E)bug message':
+                possible_actions.insert(i, 'E')
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: why are you not going for D? isnt "D" more intuitive for Debug rather than "E"?

File: coalib/output/ConsoleInteraction.py [hunk 6]
```diff
@@ -566,21 +566,31 @@ def choose_action(console_printer, actions):
...
+        possible_actions = []
         console_printer.print(format_lines('(D)o nothing'))
+        possible_actions.append('D')
         for i, action in enumerate(actions, 1):
-            console_printer.print(format_lines('{:>2}: {}'.format(
-                i,
+            console_printer.print(format_lines('{}'.format(
                 action.desc)))
-
+            if action.desc == '(O)pen file':
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: also, wouldnt it be better to use a dict? such as:
dict['O'] = false // if not applicable
dict['O'] = true // if applicable
and youd just make them all false at the start of the loop, and instead of inserting, youd just set them true :) hows that sound?

File: coalib/settings/ConfigurationGathering.py [hunk 7]
```diff
@@ -59,17 +59,18 @@ def load_config_file(filename, log_printer, silent=False):
...
-                                 'your current options by adding the `--save` '
-                                 'flag or suppress any use of config '
-                                 'files with `-I`.'
-                                 .format(Constants.default_coafile))
+                log_printer.warn('{0!r} was not found.\n'
+                                 'You can:\tGenerate a configuration file with '
+                                 'your current options by adding `--save`\n'
+                                 '        \tSuppress any use of config files'
+                                 'with `-I`.'
+                                 .format(filename))
```
Comment (Adrianzatreanu) [inline coalib/settings/ConfigurationGathering.py]: is this related to this commit or?

File: coalib/output/ConsoleInteraction.py (hunk 5 above)
Comment (Nosferatul) [inline coalib/output/ConsoleInteraction.py]: yes, but we have (D)o nothing

File: coalib/output/ConsoleInteraction.py (hunk 6 above)
Comment (Nosferatul) [inline coalib/output/ConsoleInteraction.py]: Hmm the idea is that in the end, the function needs to return an index based on what user choosed from the available actions and I don't think I can do that if I have all actions in the dict, we don't know the order of the actions. We can use a dict to just store the keys but I think it will be to much to use a dict for that. I think I can use just add instead of insert for optimization.

File: coalib/settings/ConfigurationGathering.py (hunk 7 above)
Comment (Nosferatul) [inline coalib/settings/ConfigurationGathering.py]: No

File: coalib/output/ConsoleInteraction.py (hunk 6 above)
Comment (Nosferatul) [inline coalib/output/ConsoleInteraction.py]: it works

Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: okay if you think its better like that then im with you

File: coalib/results/result_actions/PrintDebugMessageAction.py [hunk 8]
```diff
@@ -15,7 +15,7 @@ def is_applicable(result: Result, original_file_dict, file_diff_dict):
 
     def apply(self, result, original_file_dict, file_diff_dict):
         """
-        Print debug message
+        Print D(E)bug message
```
Comment (Adrianzatreanu) [inline coalib/results/result_actions/PrintDebugMessageAction.py]: i think its the 3rd time im asking and i cant see my comment anymore: why is this not "D"?

Comment (Nosferatul) [inline coalib/results/result_actions/PrintDebugMessageAction.py]: I answerd upper, we have (D)o nothing

File: coalib/output/ConsoleInteraction.py (hunk 1 above)
Comment (meetmangukiya) [inline coalib/output/ConsoleInteraction.py]: was a tuple intended? If yes, comma required, if no, get rid of brackets

**Comment by Nosferatul** [comment]
I'm closing this, work in progress: https://github.com/coala/coala/pull/4346
